protos/fabric.proto

```diff
@@ -107,10 +107,8 @@ message BlockchainInfo {
 // the block hash when verifying the blockchain.
 // localLedgerCommitTimestamp - The time at which the block was added
 // to the ledger on the local peer.
-// transactionResults - The results of transactions.
 message NonHashData {
     google.protobuf.Timestamp localLedgerCommitTimestamp = 1;
-    repeated TransactionResult transactionResults = 2;
```

Comment:
Do you mean the type? See core/ledger/ledger.go:180. Btw https://github.com/hyperledger/fabric/pull/1754 (events) removes it as far as I remember because it sends events per Tx.